### PR TITLE
Enable and Disable Addins

### PR DESCRIPTION
**Pull request by GouriKumari**
Fixes  https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1156791

Tasks Done:

This PR includes changes to enable and disable addins before starting VSFMac IDE.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/mono/mono-addins?pullRequest=162) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/mono/mono-addins?pullRequest=162) before we can accept your contribution.<br/><br/>:x: GouriKumari [sign now](https://cla.dotnetfoundation.org/mono/mono-addins?pullRequest=162)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/mono/mono-addins?pullRequest=162) it.</sub>

File: Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs [hunk 1]
```diff
@@ -395,6 +395,32 @@ void UpdateAvailableAddins (string[] args)
...
+			}
+		}
+
+		void DisableAddins(string[] args)
+		{
+			var addins = args.Where(a => a != "-y");
+			if (addins == null)
+			{
+				return;
+			}
```
Comment (Therzok) [inline Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs:418]: `addins` can't be null here

```suggestion
```

File: Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs [hunk 2]
```diff
@@ -395,6 +395,32 @@ void UpdateAvailableAddins (string[] args)
...
 			service.Repositories.UpdateAllRepositories (new ConsoleProgressStatus (verbose));
 		}
+
+		void EnableAddins (string[] args)
+		{
+			var addins = args.Where(a => a != "-y");
+			if(addins == null)
+			{
+				return;
+			}
```
Comment (Therzok) [inline Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs:405]: Same here.

```suggestion
```

File: Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs [hunk 3]
```diff
@@ -1057,6 +1075,18 @@ void CreateCommands ()
 			cmd.Description = "Lists available add-in updates.";
 			cmd.AppendDesc ("Prints a list of available add-in updates in the registered repositories.");
 			commands.Add (cmd);
+
+			cmd = new SetupCommand(cat, "enable", "e", new SetupCommandHandler (EnableAddins));
+			cmd.Description = "Enables addins.";
+			cmd.Usage = "<id> ...";
+			cmd.AppendDesc("Enables an add-in repository which has been disabled");
```
Comment (Therzok) [inline Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs:1082]: This should not have repository in the description. It disables/enables addins. Same below

**Pull request merged**